include/wx/qt/private/treeitemfactory.h

```diff
@@ -0,0 +1,123 @@
+/////////////////////////////////////////////////////////////////////////////
+// Name:        wx/treeitemfactory.h
+// Purpose:     Factory to create text edit controls for the tree items
+// Author:      Graham Dawes
+// Modified by:
+// Created:     02/2019
+// Copyright:   Graham Dawes
+// Licence:     wxWindows licence
+/////////////////////////////////////////////////////////////////////////////
+
+#ifndef _WX_TREEITEM_FACTORY_H_
```

Comment:
This convention is not followed everywhere, but normally header guards should include the path:
```suggestion
#ifndef _WX_QT_PRIVATE_TREEITEM_FACTORY_H_
```